app/assets/javascripts/pageflow/ui/views/table_cells/text_table_cell_view.js

```diff
@@ -8,6 +8,13 @@ pageflow.TextTableCellView = pageflow.TableCellView.extend({
   className: 'text_table_cell',
 
   update: function() {
-    this.$el.text(this.attributeValue());
+    if(this.attributeValue() === undefined ||
+       this.attributeValue() === null ||
+       this.attributeValue() === '') {
+      this.$el.text(this.options.column.default);
+    }
+    else {
+      this.$el.text(this.attributeValue());
+    }
```

Comment:
This is equivalent to `this.$el.text(this.attributeValue() || this.options.column.default);`. Please add a spec for this feature.